### ISSUE TITLE
VictoryBar: Missing Bar chart values for some VictoryBar values

### ISSUE THREAD

**Issue by scottdickerson** [FormidableLabs/victory#1175]
# Bugs and Questions

### Checklist

* [x] This is not a `victory-native` specific issue. (Issues that only appear in `victory-native` should be opened [here](https://github.com/FormidableLabs/victory-native/issues/new))

* [x] I have read through the [FAQ](https://formidable.com/open-source/victory/docs/faq) and [Guides](https://formidable.com/open-source/victory/guides/) before asking a question

* [x] I am using the latest version of Victory

* [x] I've searched open issues to make sure I'm not opening a duplicate issue

### The Problem

I have a VictoryStack with VictoryBars, and for some reason, it's failing to graph a few of the actual bar values:

![image](https://user-images.githubusercontent.com/6663002/48098062-3ea9b280-e1e1-11e8-883e-6d72dc8f08db.png)

Notice in the first two bars there are missing values midway down the bar that should be there...

### Reproduction

You can see the error here:
https://codesandbox.io/s/ryq0r4r0l4

**Comment by boygirl** [comment]
@scottdickerson 

Super weird. I simplified your sandbox a bit, and that seemed to help: https://codesandbox.io/s/ryp5j5zvxm.

**Comment by scottdickerson** [comment]
@boygirl Unfortunately it still is missing some data points, it just looks a little better due to the render order changing to put the missing bars at the top rather than the bottom.  Notice that at this timestamp: 11/5/2018 5:00 pm, there should be two bars shown, one for Antenna_26312_2, and one for Antenna_26312_6, but only one is shown.

```
metricValue: "Antenna_26312_2",
        data: [
          { x: "11/5/2018 5:00 pm", y: 32 },
```

```
 {
        metricValue: "Antenna_4376_6",
        data: [
          { x: "11/5/2018 4:00 pm", y: 60 },
          { x: "11/5/2018 5:00 pm", y: 60 },
```

![image](https://user-images.githubusercontent.com/6663002/48207613-a7567380-e336-11e8-9e06-761abbd0523b.png)

**Comment by kiranvenkatesh** [comment]
I am having a similar issue with the VictoryZoomContainer. First and last data points are missing along with some random data points in between.

**Comment by scottdickerson** [comment]
here's the simplest possible code pen to reproduce, notice the missing bar on the left:
https://codesandbox.io/s/5y5yyo7wqp

**Comment by boygirl** [comment]
released as `victory@31.1.0`
